tket2-py/tket2/ops.py

```diff
@@ -6,27 +6,69 @@
 class Tk2Op(Enum):
     """A Tket2 built-in operation."""
 
-    H = auto()
-    CX = auto()
-    T = auto()
-    S = auto()
     X = auto()
     Y = auto()
     Z = auto()
+    H = auto()
+    T = auto()
     Tdg = auto()
+    S = auto()
     Sdg = auto()
-    ZZMax = auto()
-    Measure = auto()
-    RzF64 = auto()
+    SX = auto()
+    SXdg = auto()
+    V = auto()
+    Vdg = auto()
     RxF64 = auto()
+    RyF64 = auto()
```

Comment:
This is not related to this PR, but I wonder why we added the "F64" suffix to Rx, Ry and Rz, but not to other operations that have angle parameters (e.g. U1).

(I actually prefer without, for conciseness.)

Reply:
Yes, I was going to propose dropping the suffix in another PR